examples/kafka_filter/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.4'
+services:
+  fluent-bit:
+      build:
```

Comment:
This could be just `image: ghcr.io/fluent/fluent-bit/pr-4641:x86_64-latest` if it defaulted to supporting the input plugin during the build.